sceptre/cli/diff.py

```diff
@@ -22,10 +26,16 @@
     help='The type of differ to use. Use "deepdiff" for recursive key/value comparison. "difflib" '
          'produces a more traditional "diff" result. Defaults to deepdiff.'
 )
+@click.option(
+    '--nonzero',
+    is_flag=True,
+    help="If this flag is passed, this will return the number of stacks with a difference as the "
+         "exit code. Otherwise, this will always return a 0 exit code except in the case of errors."
+)
```

Comment:
You know what, you've convinced me here. I'm just going to pull this out. It doesn't really provide any immediate benefit to me and I agree that it does make output a little confusing. If it's requested in the future, it would be easy to bring back.